### PR TITLE
Dbr 341 header fixes

### PR DESCRIPTION
**Pull request by cbernat**
- Add fixes for client manager when inactive section, so logo can be seen in shopify

image

- Also add client manager user json, for quick testing

Related to https://github.com/FromDoppler/Doppler-UI-System/pull/129

https://cdn.fromdoppler.com/doppler-webapp/build942/#/integrations/shopify

### CODE REVIEW

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -17,11 +17,8 @@ const Header = ({
       <header
         className={
           'header-main' +
-          (!inactiveSection
-            ? user.clientManager
-              ? ' header-open dp-header--cm'
-              : ' header-open'
-            : '')
+          (inactiveSection ? ' ' : ' header-open') +
+          (user.clientManager ? ' dp-header--cm' : ' ')
```
Comment (andresmoschini) [inline src/components/Header/Header.js]: 👍

**Comment by fcoronelmakingsense** [comment]
its need the other .json?

**Comment by cbernat** [comment]
> its need the other .json?

Yes for testing, whenever you need to test a cm user, just copy and paste into headerData.json, we could add more files for specific and rare cases of users.

File: src/dopplerUserJson/cmUser.json [hunk 2]
```diff
@@ -0,0 +1,194 @@
+{
```
Comment (andresmoschini) [inline src/dopplerUserJson/cmUser.json]: What do you think about renaming it in a more descriptive way?

For example: `/src/headerData-for-client-manager-user.json `

Comment (cbernat) [reply src/dopplerUserJson/cmUser.json]: Yes it sounds more understandable.

Comment (cbernat) [reply src/dopplerUserJson/cmUser.json]: I've also remembered YAGNI principle.

Comment (andresmoschini) [reply src/dopplerUserJson/cmUser.json]: Just curiosity, how does YAGNI apply here?

Comment (cbernat) [reply src/dopplerUserJson/cmUser.json]: The folder, may not be needed.

**Pull request merged**